core/DataAccess/Model.php

```diff
@@ -377,7 +377,7 @@ public function deleteOlderArchives(Parameters $params, $name, $tsArchived, $idA
 
         $sql = "SELECT idarchive FROM `$numericTable` WHERE idsite = ? AND date1 = ? AND date2 = ? AND period = ? AND name = ? AND ts_archived < ? AND idarchive < ?";
 
-        $idArchives = Db::fetchAll($sql, [$params->getSite()->getId(), $dateStart, $dateEnd, $params->getPeriod()->getId(), $name, $tsArchived, $idArchive]);
+        $idArchives = Db::fetchAll($sql, [$params->getSite()->getId(), $dateStart . '', $dateEnd . '', $params->getPeriod()->getId(), $name, $tsArchived, $idArchive]);
```

Comment:
Would it be good to use `->getDatetime()`? Otherwise good to merge.

Reply:
👍  Good one. So maybe partially it could have been buggy before?

Reply:
Maybe, I remember testing this and it working at least on my local setup...

Reply:
@diosmosis updated it